tests/constants/govc/cli.go

```diff
@@ -34,4 +35,10 @@ const (
 
 	// VirtualMachineName JSON object key refers VM name
 	VirtualMachineName = govcResponseRoot + ".Name"
```

Comment:
Let's keep the naming consistent: since we are using the abbr "VM" above, let's also use the abbr for these two:
VirtualMachineName => VMName
VirtualMachinePowerState => VMPowerState